ds-cv-inference/Dockerfile

```diff
@@ -28,6 +28,7 @@ COPY go.* /go/src/ds-cv-inference/
 RUN go mod download
```

Comment:
if we already do the go mod tidy below, do we still need this go mod download?

Reply:
yup. I removed it and forgot to push the change.